content/python/concepts/errors/errors.md

```diff
@@ -18,7 +18,7 @@ There are (at least) two distinguishable kinds of errors in Python: syntax error
 
 Syntax errors, also known as parsing errors, are perhaps the most common kind of complaint you get while you are still learning Python:
```

Comment:
```suggestion
Syntax errors (also known as parsing errors) occur when a sequence of characters, or tokens, violates the syntax of the Python programming language:
```

For the rest of this entry, let's replace all instances of "you" with a neutral, more manual-like tone that doesn't address the reader(s) directly.

Reply:
Also, on line # 15, can we change to make it look similar to the `Description` on line # 3. Also, since "errors" is the entry concept, let's place it **in bold**. Maybe something like this:

"The two types of **errors** are syntax errors and exceptions."